AsyncDisplayKit/ASTableView.mm

```diff
@@ -455,6 +455,14 @@ - (void)reloadRowsAtIndexPaths:(NSArray *)indexPaths withRowAnimation:(UITableVi
   [_dataController reloadRowsAtIndexPaths:indexPaths withAnimationOptions:animation];
 }
 
+- (void)relayoutRowAtIndexPath:(NSIndexPath *)indexPath withRowAnimation:(UITableViewRowAnimation)animation
+{
+  ASDisplayNodeAssertMainThread();
+  ASCellNode *node = [self nodeForRowAtIndexPath:indexPath];
+  [node setNeedsLayout];
```

Comment:
I guess this also brings up the question of how a developer could force display of the new destination positions.  If we do layoutIfNeeded, then they could call -displayImmediately on the applicable nodes right afterwards; otherwise they would have to rely on toggling isSynchronous, which is cumbersome since it doesn't reset itself after the next display...